ci_framework/roles/dlrn_promote/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install dlrnapi-client
+  ansible.builtin.import_tasks: install.yml
+
+- name: Get hash related data from repo_setup role
+  ansible.builtin.include_role:
+    name: repo_setup
+
+- name: Check if commit hash is already promoted to target name
+  ansible.builtin.import_tasks: check_for_previous_promotions.yml
+  when: cifmw_dlrn_promote_hash | bool
+
+- name: Run DLRN promote hash
+  when:
+     - cifmw_dlrn_promote_hash_in_promote_target is defined
+     - cifmw_dlrn_promote_hash_in_promote_target.stdout == '[]'
+  block:
+    - name: Check reported jobs
+      ansible.builtin.import_tasks: check_reported_jobs.yml
+
+    - name: Check promotion criteria
+      ansible.builtin.import_tasks: check_promotion_criteria.yml
+
+    - name: Promote {{ cifmw_dlrn_promote_promotion_source }} to {{ cifmw_dlrn_promote_promotion_target }}
+      ansible.builtin.import_tasks: promote_hash.yml
```

Comment:
this playbook is missing that's why the pre_commit is failing. I need to add that file. https://github.com/openstack-k8s-operators/ci-framework/pull/558/commits/749f1ce581cbbc2522ec904c34ef4ae6e57e30b7#diff-015df08b3ca14fa93991f1de82509e25c6a082ba20fa908dd311994e035afa7cR40

Reply:
resolve the comment if this is now done? (I think it is)